src/System.Collections.NonGeneric/tests/HashtableTests.cs

```diff
@@ -1415,36 +1414,20 @@ public override int GetHashCode()
                 return 0;
             }
 
-            public override string ToString()
-            {
-                return Value.ToString();
-            }
+            public override string ToString() => Value.ToString();
         }
 
         private class Foo
         {
-            private string _stringValue = "Hello World";
-            public string StringValue
-            {
-                get { return _stringValue; }
-                set { _stringValue = value; }
-            }
+            public string StringValue { get; set; } = "Hello World";
```

Comment:
This is new c# 6 stuff - its quite neat in reducing boilerplate and writing clearer code